### PR TITLE
Add scrollbar to MFA screen

### PR DESCRIPTION
**Pull request by stsai-okta**
Resolves: OKTA-84391

### CODE REVIEW

**Comment by stsai-okta** [comment]
https://okta.box.com/s/dkp2ov81g3y4gwv34n6ay4ib3vbb5s0l

**Comment by Ujjwalreddy-okta** [comment]
Just checking, did we test this on IE browsers and mobile?

File: assets/sass/_ie.scss [hunk 1]
```diff
@@ -12,6 +12,16 @@
     }
   }
 
+  .auth {
+    max-height: 500px;
+    overflow: auto;
+  }
+
+  .footer {
```
Comment (jhe-okta) [inline assets/sass/_ie.scss]: I am not familiar with the code. What is the footer class for?

**Comment by stsai-okta** [comment]
tested on IE and it looks like it works. need to test on mobile still

File: assets/sass/_ie.scss (hunk 1 above)
Comment (stsai-okta) [inline assets/sass/_ie.scss]: the footer on the loginpage is 
image

if we don't add the footer class, the footer shows up in the middle of the widget which looks funny

Comment (jhe-okta) [inline assets/sass/_ie.scss]: Looks like it should be position fixed to the bottom of the page.

Comment (stsai-okta) [inline assets/sass/_ie.scss]: updated

**Comment by stsai-okta** [comment]
updated footer css and tested on my iphone @Ujjwalreddy-okta @jhe-okta

**Comment by jhe-okta** [comment]
🍧

**Comment by Ujjwalreddy-okta** [comment]
🚀

**Pull request merged**